server/services/__tests__/bodyProfile.test.ts

```
@@ -0,0 +1,237 @@
+      }
+
+      const mockUser = { id: userId }
+      const mockBodyProfile = {
+        id: 'profile-id',
+        userId,
+        ...profileData,
+        createdAt: new Date(),
+        updatedAt: new Date(),
+      }
+
+      jest.spyOn(bodyProfileService['userRepository'], 'findByAuth0Id').mockResolvedValue(mockUser as any)
```

Biome (1.9.4)

[error] The computed expression can be simplified without the use of a string literal.

Unsafe fix: Use a literal key instead.


(lint/complexity/useLiteralKeys)